sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -133,8 +143,15 @@ def read(
         """
         request_options = build_options(kwargs)
         response_hook = kwargs.pop('response_hook', None)
-        if populate_query_metrics is not None:
+        populate_query_metrics = args[0] if args else None or kwargs.pop('populate_query_metrics', None)
```

Comment:
I would also recommend adding some tests here to validate the non-breaking behaviour for both positional and keyword parameters.